jobclient/src/main/java/com/twosigma/cook/jobclient/JobClient.java

```diff
@@ -1029,12 +1018,13 @@ public void abort(Collection<UUID> uuids)
                 URIBuilder uriBuilder = new URIBuilder(_jobURI);
                 uriBuilder.addParameters(params);
                 httpRequest =  new HttpDelete(uriBuilder.build());
+                addImpersonation(httpRequest, impersonatedUser);
             } catch (URISyntaxException e) {
                 throw releaseAndCreateException(null, null, "Can not submit DELETE request " + params + " via uri " + _jobURI, e);
             }
             HttpResponse httpResponse;
             try {
-                httpResponse = _httpClient.execute(httpRequest);
+                httpResponse = executeWithRetries(httpRequest, 5, 10);
```

Comment:
Is this change related to impersonation?

Reply:
Yes, because delegating to `executeWithRetries` means I have to handle injecting the impersonation headers in fewer places.